nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -475,6 +475,8 @@ public Description matchAssignment(AssignmentTree tree, VisitorState state) {
           state,
           ASTHelpers.getSymbol(tree.getVariable()));
     }
+    handler.handleFieldAssignment(
```

Comment:
This will only run for assignments to fields where there is no `@Nullable` annotation and also no error (since the method would have returned already), which seems a little bit weird for the general case.  Is that what you want?  Or should we move this call up to around line 463?

Reply:
Quick summary of our discussion, since for our final purpose of finding intializer methods, we need this method to get called when a method is actually writing a `@Nonnull` value to a `@Nonnull` field and calling `mayBeNullExpr` is costly, we decided to rename the method to match the current situation.

5329e44533eeea80189516e4ac6289f13578766f